server/chat-commands/core.ts

```diff
@@ -1583,6 +1583,11 @@ export const commands: ChatCommands = {
 			this.errorReply(`Could not find help for '/${target}' - displaying help for '/${currentBestHelp.for.join(' ')}' instead`);
 		}
 
+		const curHandler = this.parseCommand(`/${currentBestHelp.for[0]}`)?.handler;
+		if (curHandler?.isPrivate && !user.can('lock')) {
```

Comment:
I wonder if putting it on the `lock` permission is bad here. Maybe the command should specify permission.